### PR TITLE
[resumes][feat] Add submit form

### PR DESCRIPTION
**Pull request by keanecjy**

### CODE REVIEW

File: apps/portal/src/pages/_app.tsx [hunk 1]
```diff
@@ -11,8 +11,8 @@ import AppShell from '~/components/global/AppShell';
 
 import type { AppRouter } from '~/server/router';
 
-import '~/styles/globals.css';
```
Comment (yangshun) [inline apps/portal/src/pages/_app.tsx:14]: Did this ordering cause any bug?

Comment (keanecjy) [reply apps/portal/src/pages/_app.tsx:14]: I saw the sidebar menu disappear with this ordering, is that intended? 
image

Comment (yangshun) [reply apps/portal/src/pages/_app.tsx:14]: Oh wow definitely not intended. Thanks for fixing :)

Comment (keanecjy) [reply apps/portal/src/pages/_app.tsx:14]: Strange thing is, the button disappears when I change the ordering and only appears onHover


https://user-images.githubusercontent.com/65328119/194038933-085d2a0c-8337-4f5d-b2ae-8915e6ac1f09.mov

**Pull request closed without merging**